content/operate/kubernetes/reference/yaml/rack-awareness.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: Rack awareness examples
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: YAML examples for rack-aware Redis Enterprise deployments that distribute nodes across availability zones.
+linkTitle: Rack awareness
+weight: 20
+---
+
+This page provides YAML examples for deploying Redis Enterprise with [rack awareness]({{< relref "/operate/kubernetes/architecture/operator-architecture#rack-awareness" >}}). Rack awareness distributes Redis Enterprise nodes across different availability zones or failure domains to improve high availability and fault tolerance.
```

Comment:
```suggestion
This page provides YAML examples for deploying Redis Enterprise with [rack awareness]({{< relref "/operate/kubernetes/architecture/operator-architecture#rack-awareness" >}}). Rack awareness distributes Redis Enterprise nodes and database shards across different availability zones or failure domains to improve high availability and fault tolerance.
```

May want to mention the effect on databases too, which I guess is the more important aspect for most users.